.github/workflows/workspace-integration-tests.yml

```diff
@@ -58,19 +67,19 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          if [[ '${{ github.event.inputs.name }}' != '' ]]; then
+          if [[ '${{ inputs.name }}' != '' ]]; then
               {
-                  echo "name=${{ github.event.inputs.name }}"
+                  echo "name=${{ inputs.name }}"
               } >> $GITHUB_OUTPUT
           else
               {
                   echo "name=workspace-integration-test-${{ github.run_id }}-${{ github.run_attempt }}"
               } >> $GITHUB_OUTPUT
           fi
 
-          if [[ '${{ github.event.inputs.version }}' != '' ]]; then
+          if [[ '${{ inputs.version }}' != '' ]]; then
```

Comment:
for `workflow_call` `github.event.inputs.*` doesnt exist and need to use `inputs.*` instead. Works now: https://github.com/gitpod-io/gitpod/actions/runs/5692256844/job/15429016661?pr=18381#step:6:12

`inputs.*` is also available for `workflow_dispatch`